lib/managers/docker.js

```diff
@@ -804,36 +807,17 @@ export const extractTar = async (fullImageName, dir) => {
         "Please run cdxgen from a powershell terminal with admin privileges to create symlinks.",
       );
       console.log(err);
-    } else if (
-      ![
-        "TAR_BAD_ARCHIVE",
-        "TAR_ENTRY_INFO",
-        "TAR_ENTRY_INVALID",
-        "TAR_ENTRY_ERROR",
-        "TAR_ENTRY_UNSUPPORTED",
-        "TAR_ABORT",
-        "EACCES",
-      ].includes(err.code)
-    ) {
-      console.log(
-        `Error while extracting image ${fullImageName} to ${dir}. Please file this bug to the cdxgen repo. https://github.com/CycloneDX/cdxgen/issues`,
-      );
-      console.log("------------");
-      console.log(err);
-      console.log("------------");
     } else if (err.code === "TAR_BAD_ARCHIVE") {
       if (DEBUG_MODE) {
         console.log(`Archive ${fullImageName} is empty. Skipping.`);
       }
-    } else if (["EACCES"].includes(err.code)) {
```

Comment:
Oops, I did miss this one.  This should really be handled separately since you don't want a bug report for it.

Reply:
If it helps, here is a summary of how all those tests should shake out now:


| **Code**              | **Old**                                                                       | **New**                                                                       |
| --------------------- | ----------------------------------------------------------------------------- | ----------------------------------------------------------------------------- |
| TAR_BAD_ARCHIVE       | Prints  "Archive ${fullImageName} is empty. Skipping." and ends extraction.   | Prints "Archive ${fullImageName} is empty. Skipping." and ends extraction.    |
| TAR_ENTRY_INFO        | Black holed without any message and ends extraction.                          | Prints warning only if `DEBUG_MODE` is `true` and continues extraction.       |
| TAR_ENTRY_INVALID     | Prints `err` and ends extraction.                                             | Prints warning and continues extraction.                                      |
| TAR_ENTRY_ERROR       | Prints `err` and ends extraction.                                             | Prints warning and continues extraction.                                      |
| TAR_ENTRY_UNSUPPORTED | Prints `err` and ends extraction.                                             | Prints warning and continues extraction.                                      |
| TAR_ABORT             | Prints `err` and ends extraction.                                             | Prints "Error while extracting image..." and ends extraction.                 |
| EACCES                | Prints `err` and ends extraction.                                             | Prints `err` and ends extraction.                                             |
| EPERM                 | Prints "Please run cdxgen from a powershell terminal..." and ends extraction. | Prints "Please run cdxgen from a powershell terminal..." and ends extraction. |
| Anything unexpected   | Prints `err` and ends extraction.                                             | Prints "Error while extracting image..." and ends extraction.                 |